editor/editor_vcs_api.h

```diff
@@ -0,0 +1,25 @@
+#ifndef EDITOR_VCS_API_H
+#define EDITOR_VCS_API_H
+
+#include "core/ustring.h"
+#include "core/object.h"
+
+class EditorVCSAPI {
+
+private:
+	String vcs_name;
+
+	static void register_to_editor(String vcs_name);
```

Comment:
We usually prefix all private function with an underscore, like you did with the other ones. :)